src/main/java/org/mule/tools/apikit/output/GenerationModel.java

```diff
@@ -191,7 +211,7 @@ public String getFlowName() {
         .append(FLOW_NAME_SEPARATOR)
         .append(resource.getResolvedUri(version));
 
-    if (mimeType != null) {
+    if (mimeType != null && shouldIncludeMimeTypeInName) {
```

Comment:
when should not add the mimeType?

Reply:
https://github.com/mulesoft/apikit-rest-scaffolder/pull/10#discussion_r303126042